swri_image_util/src/blend_images_util.cpp

```diff
@@ -67,4 +69,91 @@ namespace swri_image_util
 
     cv::addWeighted(base_image, 1.0 - alpha, top_image, alpha, 0, dest_image);
   }
+
+  void blendImages(
+      const cv::Mat& base_image,
+      const cv::Mat& top_image,
+      const double alpha,
+      const cv::Scalar mask_color,
+      cv::Mat& dest_image)
+  {
+    const bool debug = false;
```

Comment:
If this debugging code is something that's still useful to have in here, I'd rather this be a parameter so it can be turned on or off at runtime.  If it's not still useful, it should probably be removed.